src/legacy/core_plugins/metric_vis/public/metric_vis_fn.js

```diff
@@ -32,21 +32,114 @@ export const metric = () => ({
     defaultMessage: 'Metric visualization'
   }),
   args: {
-    visConfig: {
-      types: ['string', 'null'],
-      default: '"{}"',
+    percentage: {
+      types: ['boolean'],
+      default: false,
+      help: i18n.translate('metricVis.function.percentage.help', {
+        defaultMessage: 'Shows metric in percentage mode. Dont forget to set colorRange.'
+      })
+    },
+    colorSchema: {
+      types: ['string'],
+      default: '"Green to Red"',
+      help: i18n.translate('metricVis.function.colorSchema.help', {
+        defaultMessage: 'Color schema to use'
+      })
+    },
+    colorMode: {
+      types: ['string'],
+      default: '"None"',
+      options: ['None', 'Label', 'Background'],
+      help: i18n.translate('metricVis.function.colorMode.help', {
+        defaultMessage: 'Which part of metric to color'
+      })
+    },
+    colorRange: {
+      types: ['string'],
+      default: '"[{ from: 0, to: 10000 }]"',
+      help: i18n.translate('metricVis.function.colorRange.help', {
+        defaultMessage: 'Color ranges'
+      })
+    },
+    useRanges: {
+      types: ['boolean'],
+      default: false,
+    },
+    invertColors: {
+      types: ['boolean'],
+      default: false,
+    },
+    showLabels: {
+      types: ['boolean'],
+      default: true,
+    },
+    bgFill: {
+      types: ['string'],
+      default: '"#000"',
```

Comment:
Help text would probably be useful here to indicate this is a hex code

Reply:
nit: `aliases: ['backgroundFill', 'bgColor', 'backgroundColor']` would be nice too... if my assumption about what this does is right anyway.